pkg/controllers/generic/import_syncer.go

```diff
@@ -143,6 +142,10 @@ func (s *importer) Syncer() syncertypes.Sync[client.Object] {
 	return syncer.ToGenericSyncer[*unstructured.Unstructured](s)
 }
 
+func (s *importer) Migrate(_ *synccontext.RegisterContext, _ synccontext.Mapper) error {
```

Comment:
what is this for ?

Reply:
Migrate replaces basically the index we had before. Since we startup without "knowing" anymore what objects were how translated, we have the migrate function that initially populates the store with the virtual <-> host mappings